devdoc/object_lifetime_tracker.md

```diff
@@ -38,17 +38,28 @@ MU_DEFINE_ENUM(OBJECT_LIFETIME_TRACKER_REGISTER_OBJECT_RESULT, OBJECT_LIFETIME_T
 
 MU_DEFINE_ENUM(OBJECT_LIFETIME_TRACKER_UNREGISTER_OBJECT_RESULT, OBJECT_LIFETIME_TRACKER_UNREGISTER_OBJECT_RESULT_VALUES);
 
+#define OBJECT_LIFETIME_TRACKER_ACT_RESULT_VALUES \
+    OBJECT_LIFETIME_TRACKER_ACT_OK, \
+    OBJECT_LIFETIME_TRACKER_ACT_ERROR, \
+    OBJECT_LIFETIME_TRACKER_ACT_NOT_FOUND, \
+    OBJECT_LIFETIME_TRACKER_UNREGISTER_KEY_NOT_FOUND
```

Comment:
>OBJECT_LIFETIME_TRACKER_UNREGISTER_KEY_NOT_FOUND [](http://example.com/codeflow?start=4&length=48)

I find it weird to have `OBJECT_LIFETIME_TRACKER_UNREGISTER_KEY_NOT_FOUND` as an enum value in an enum called `OBJECT_LIFETIME_TRACKER_ACT_RESULT`. Is this intended?

Reply:
Fixed.